sdk/data/aztables/README.md

```diff
@@ -1,151 +1,348 @@
+
+### Clients
+Two different clients are provided to interact with the various components of the Table Service:
+1. **`ServiceClient`** -
+    * Get and set account setting
+    * Query, create, and delete tables within the account.
+    * Get a `Client` to access a specific table using the `NewClient` method.
+2. **`Client`** -
+    * Interacts with a specific table (which need not exist yet).
+    * Create, delete, query, and upsert entities within the specified table.
+    * Create or delete the specified table itself.
+
+### Entities
+Entities are similar to rows. An entity has a **`PartitionKey`**, a **`RowKey`**, and a set of properties. A property is a name value pair, similar to a column. Every entity in a table does not need to have the same properties. Entities are returned as JSON, allowing developers to use JSON marshalling and unmarshalling techniques. Additionally, you can use the `aztables.EDMEntity` to ensure proper round-trip serialization of all properties.
+```golang
+aztables.EDMEntity{
+    Entity: aztables.Entity{
+        PartitionKey: "pencils",
+        RowKey: "id-003",
+    },
+    Properties: map[string]interface{}{
+        "Product": "Ticonderoga Pencils",
+        "Price": 5.00,
+        "Count": aztables.EDMInt64(12345678901234),
+        "ProductGUID": aztables.EDMGUID("some-guid-value"),
+        "DateReceived": aztables.EDMDateTime(time.Date{....}),
+        "ProductCode": aztables.EDMBinary([]byte{"somebinaryvalue"})
+    }
+}
+```
+
+## Examples
+
+The following sections provide several code snippets covering some of the most common Table tasks, including:
+
+* [Creating a table](#creating-a-table "Creating a table")
+* [Creating entities](#creating-entities "Creating entities")
+* [Listing entities](#querying-entities "Listing entities")
+
+
+### Creating a table
+Create a table in your account and get a `Client` to perform operations on the newly created table:
+
+```golang
+cred, err := aztables.NewSharedKeyCredential("myAccountName", "myAccountKey")
+handle(err)
+service, err := aztables.NewServiceClient("https://<my_account_name>.table.core.windows.net", cred, nil)
+handle(err)
+resp, err := service.CreateTable("myTable")
+```
+
+### Creating entities
+Create entities in the table:
+
+```golang
+cred, err := aztables.NewSharedKeyCredential("myAccountName", "myAccountKey")
+handle(err)
+service, err := aztables.NewServiceClient("https://<my_account_name>.table.core.windows.net", cred, nil)
+handle(err)
+
+myEntity := aztables.EDMEntity{
+    Entity: aztables.Entity{
+        PartitionKey: "001234",
+        RowKey: "RedMarker",
+    },
+    Properties: map[string]interface{}{
+        "Stock": 15,
+        "Price": 9.99,
+        "Comments": "great product",
+        "OnSale": true,
+        "ReducedPrice": 7.99,
+        "PurchaseDate": aztables.EDMDateTime(time.Date(2021, time.August, 21, 1, 1, 0, 0, time.UTC)),
+        "BinaryRepresentation": aztables.EDMBinary([]byte{"Bytesliceinfo"})
```

Comment:
It's strange to save a string as binary; I'd use a string.
If you want to show EDMBinary, then use bytes